pandas/core/internals/construction.py

```diff
@@ -511,29 +513,32 @@ def to_arrays(data, columns, coerce_float=False, dtype=None):
         return _list_to_arrays(data, columns, coerce_float=coerce_float, dtype=dtype)
 
 
-def _list_to_arrays(data, columns, coerce_float=False, dtype=None):
+def _list_to_arrays(
+    data, columns, coerce_float=False, dtype=None
+) -> Tuple[list, Iterable]:
```

Comment:
```suggestion
) -> Tuple[List, Iterable]:
```

If you can add sub-types for List and Iterable would be even better